doc/source/serve/production.md

```diff
@@ -1,66 +1,63 @@
-- understand the Serve config file format
-- deploy, inspect, and update your Serve application in production
+- understand the Serve config file format.
+- understand how to deploy Ray Serve using the REST API / CLI directly.
+- understand how to deploy Ray Serve on Kubernetes using the [RayService] custom resource.
 
 ```{contents}
 ```
 
-(serve-in-production-testing)=
-
-## Developing and Testing Your Serve Deployment Graph with `serve run`
+(serve-in-production-example)=
 
-You can test your Serve deployment graph using the Serve CLI's `serve run` command. The `serve run` command launches a temporary Ray cluster, deploys the graph to it, and blocks. Then, you can send HTTP requests to test your application. When your graph receives and processes these requests, it will output `print` and `logging` statements to the terminal. Once you're finished testing your graph, you can type `ctrl-C` to kill the temporary Ray cluster and tear down your graph. You can use this pattern to quickly run, debug, and iterate on your Serve deployment graph.
+## Working example: FruitStand application
 
-Let's use this graph as an example:
+This user guide will use the following Serve application as a working example. It takes in requests containing a list of two values, a fruit name and an amount, and returns the total price for the batch of fruits.
 
 ```{literalinclude} ../serve/doc_code/production_fruit_example.py
 :language: python
 :start-after: __fruit_example_begin__
 :end-before: __fruit_example_end__
 ```
 
-This graph is located in the `fruit.py` file and stored in the `deployment_graph` variable. It takes in requests containing a list of two values: a fruit name and an amount. It returns the total price for the batch of fruits.
-
-To run the deployment graph, we first navigate to the same directory containing the `fruit.py` file and then run `serve run fruit.deployment_graph`. `fruit.deployment_graph` is the deployment graph's import path (assuming we are running `serve run` in the same directory as `fruit.py`).
+This code is saved in the `fruit.py` file and the final deployment graph that we want to deploy is assigned to the `deployment_graph` variable.
+To run and test the application locally, use the `serve run fruit:deployment_graph` command:
 
 ```console
-# Terminal Window 1
+# Terminal window 1:
 
 $ ls
 fruit.py
 
-$ serve run fruit.deployment_graph
-2022-06-21 13:07:01,966  INFO scripts.py:253 -- Deploying from import path: "fruit.deployment_graph".
+$ serve run fruit:deployment_graph
+2022-06-21 13:07:01,966  INFO scripts.py:253 -- Deploying from import path: "fruit:deployment_graph".
 2022-06-21 13:07:03,774  INFO services.py:1477 -- View the Ray dashboard at http://127.0.0.1:8265
 ...
 2022-06-21 13:07:08,076  SUCC scripts.py:266 -- Deployed successfully.
+
 ```
 
 We can test this graph by opening a new terminal window and making requests with Python's [requests](https://requests.readthedocs.io/en/latest/) library.
```

Comment:
Can we update this to discuss the `curl` command instead of the `requests` library? Alternatively, could we show how to make a request with the `requests` library?